pkg/cloudprovider/cloud.go

```diff
@@ -63,6 +63,14 @@ func GetLoadBalancerName(service *api.Service) string {
 	return ret
 }
 
+func HostsFromNodeList(list *api.NodeList) []string {
```

Comment:
**HostsFromNodeList** should probably be **hostsFromNodeList** because it is called as **hostsFromNodeList** later.